packages/now-cli/test/dev/fixtures/public-and-server-as-api/now.json

```diff
@@ -0,0 +1,7 @@
+{
+  "functions": {
+    "server/**/*.js": {
+      "runtime": "@now/node@1.2.1"
```

Comment:
I thought that using an Official Runtime such as `@now/node` would fail to build?

Reply:
@styfle not anymore, since this way we allow users to use files outside of the `api` directory, and they can lock version for now this way.